src/main/asciidoc/driver/configuration.adoc

```diff
@@ -154,7 +151,16 @@ include::{python-examples}/test_examples.py[tag=tls-trust-on-first-use]
 --
 ====
 
-_Trust signed certificate_ means that the driver will only connect to a host if it presents a trusted, signed certificate.
+_Trust signed certificate_ means that the driver will only connect to a Neo4j server if it provides a certificate trusted by the driver.
+In this scenario, one or more trusted certificate are installed in the driver.
+The server then authenticate itself to the driver with a certificate signed by one of the trusted certificates known to the driver.
+
+The way to install a trusted certificate into a driver varies for differ drivers.
+The Java driver directly accepts the file path to the trusted certificate and would load the certificate automatically during runtime (See the example bellow).
+However for .NET and Python driver, in order for the drivers to trust a certificate, the certificate need to be imported into the operation system's trusted certificate store.
```

Comment:
How about JavaScript? If the Java driver is the only one that works this way we should explain the majority case as the norm and talk about the Java driver as the exception.